src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -121,12 +127,33 @@ private void updateSensorState() {
     currentAngle = motor.getPosition().refresh().getValueAsDouble() * 2 * Math.PI;
     currentVelocity = motor.getVelocity().refresh().getValueAsDouble() * 2 * Math.PI;
     currentAbsoluteAngle = absoluteEncoder.get();
+    checkError();
 
     logCurrentAngle.append(currentAngle);
     logCurrentVelocity.append(currentVelocity);
     logCurrentAbsoluteAngle.append(currentAbsoluteAngle);
   }
 
+  /**
+   * Checks if the arm is beyond its maximum and minimum angles by 2.0 degrees or is taking more
+   * than 1.0 second to be within 2.0 degrees of the goal angle.
+   */
+  private void checkError() {
+    if (MathUtil.isNear(goalAngle,  currentAngle, ERROR_MARGIN) {
```

Comment:
Missing closing parenthesis.
```suggestion
    if (MathUtil.isNear(goalAngle,  currentAngle, ERROR_MARGIN)) {
```